Sources/Shared/LocalAuthentication/BiometricsAuthenticationViewController.swift

```diff
@@ -0,0 +1,87 @@
+import UIKit
+
+protocol BiometricsAuthenticationViewControllerDelegate: AnyObject {
+    func didTapUnlock()
+}
+
+final class BiometricsAuthenticationViewController: UIViewController {
+    private let homeAssistantLogo: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "launchScreen-logo")
+        imageView.frame = CGRect(x: 0, y: 0, width: 256, height: 256)
+        imageView.contentMode = .scaleAspectFit
+        return imageView
+    }()
+
+    private let unlockButton: UIView = {
+        let containerView = UIView()
+        let button = UIButton()
+        button.setTitle(L10n.SettingsDetails.General.Security.unlockButton, for: .normal)
+        button.titleLabel?.font = UIFont.preferredFont(forTextStyle: .headline)
```

Comment:
how does this end up getting its title color set? i don't see relevant code but maybe i missed it